frontend/apps/picasso/defi/polkadot/hooks/crowdloanRewards/useCrowdloanRewardsClaim.ts

```diff
@@ -27,7 +27,7 @@ export function useCrowdloanRewardsClaim({
       enqueueSnackbar("Claim Processing", {
         variant: "info",
         isClosable: true,
-        url: SUBSTRATE_NETWORKS.picasso.subscanUrl + transactionHash,
+        url: SUBSTRATE_NETWORKS.picasso.subscanUrl + "extrinsic/" + transactionHash,
```

Comment:
Better to create a shared function to produce these URLs for us: 
```ts
url: getSubscanURL(txHash)
```

Reply:
@easteregg for sure, lets do that